testsuite/pom.xml

```diff
@@ -231,7 +231,6 @@
                                            <exclude>bin/*.bat</exclude>
                                            <exclude>bin/*.ps1</exclude>
                                            <exclude>docs/</exclude>
-                                           <exclude>modules/</exclude>
```

Comment:
We can't do this; copying the modules all over puts too much IO strain on the CI servers.